src/proto/mgmt/storage.proto

```diff
@@ -121,6 +118,67 @@ message ScmModule {
 	//string fwrev = 10;
 }
 
+enum ResponseStatus {
+	SUCCESS = 0;
+	IN_PROGRESS = 1;
+	WAITING = 2;
+	ERR_APP = -1;			// Application error
+	ERR_UNKNOWN = -2;		// Unknown error
+}
+
+message ResponseState {
+	ResponseStatus status = 1;
+	string error = 2;
+	string info = 3;
+}
+
+message NvmeControllerResp {
+	string pciaddr = 1;		// PCI address of NVMe controller
+	ResponseState state = 2;
+}
+
+message ScmMountResp {
+	string mntpoint = 1;		// Whene device or tmpfs is mounted
```

Comment:
This structure might also benefit from including some other identifying information about the PMM. Maybe serial number? And/or information about the device location (socket/controller/slot). I imagine this is barely relevant to the user in most cases, but seems like it could be helpful if we encounter errors.

Reply:
given this will refer to a mounted device which is actually a namespace built on top of an AppDirect memory region I will probably add details related to the region rather than the individual modules. I imagine when introducing regions they will map to individual modules anyway

Reply:
Ah, right, the namespace capacity comes from all modules in the interleave set. I still think it may be useful to list the modules - if there is an error the user may want to run a diagnostic on the modules, or replace the ill-behaved one.

Reply:
agreed, going to have representations of both SCM mounts and modules